Python/fitnesse/html_util.py

```diff
@@ -2,41 +2,41 @@
 
 from fitnesse.context import SuiteResponder, PageCrawlerImpl, PageData, PathParser, WikiPage, WikiPagePath
 
-
+class TestType:
+    SETUP: "SetUp"
+    TEARDOWN: "TearDown"
 class HtmlUtil:
 
+    @staticmethod
+    def get_Path(page_name: str, wiki_page: WikiPage):
+        inherit_Page: WikiPage = page_crawler.get_inherited_page(page_name, wiki_page)
+        if inherit_Page is not None:
+            path: wiki_page.get_page_crawler().get_full_path(setup)
+            path_name: str = path_parser.render(page_path)
+            return path_name
+
     @staticmethod
     def testable_html(page_data: PageData, include_suite_setup: bool, page_crawler: PageCrawlerImpl, path_parser: PathParser) -> str:
         wiki_page: WikiPage = page_data.get_wiki_page()
         string_io: StringIO = StringIO()
 
         if page_data.has_attribute("Test"):
             if include_suite_setup:
-                suite_setup: WikiPage = page_crawler.get_inherited_page(SuiteResponder.SUITE_SETUP_NAME, wiki_page)
-                if suite_setup is not None:
-                    page_path: WikiPagePath = wiki_page.get_page_crawler().get_full_path(suite_setup)
-                    page_path_name: str = path_parser.render(page_path)
-                    string_io.writelines(["!include -setup .", page_path_name, "\n"])
-
-            setup: WikiPage = page_crawler.get_inherited_page("SetUp", wiki_page)
-            if setup is not None:
-                setup_path: WikiPagePath = wiki_page.get_page_crawler().get_full_path(setup)
-                setup_path_name: str = path_parser.render(setup_path)
-                string_io.writelines(["!include -setup .", setup_path_name, "\n"])
+                page_path_name = HtmlUtil.get_Path(wiki_page=wiki_page, page_name=SuiteResponder.SUITE_SETUP_NAME)
+                string_io.writelines(["!include -setup .", page_path_name, "\n"])
+
+            setup_path_name = HtmlUtil.get_Path(wiki_page=wiki_page, page_name=TestType.SETUP)
+            string_io.writelines(["!include -setup .", page_path_name, "\n"])
```

Comment:
Please rework this copy + paste error: this line uses "page_path_name", while the line before declares "setup_path_name"